java/jdbc-driver-graalvm-nativeimage/README.md

```diff
@@ -0,0 +1,2 @@
+# jdbc-driver-graalvm-nativeimage
+[Introduction to Oracle JDBC Driver Configuration for GraalVM Native Image](https://juarezjunior.medium.com/introduction-to-oracle-jdbc-driver-configuration-for-graalvm-native-image-6130e699d03b)
```

Comment:
Please use the Oracle blog platform.

Reply:
After our call you asked me to disregard this one as it will be targeted at at oracledevs on Medium as usual